periphboard/board.go

```diff
@@ -17,6 +17,7 @@ import (
 	"periph.io/x/conn/v3/gpio"
 	"periph.io/x/conn/v3/gpio/gpioreg"
 	"periph.io/x/conn/v3/physic"
+	"periph.io/x/host/v3"
 
 	"go.viam.com/rdk/components/board"
 	"go.viam.com/rdk/grpc"
```

Comment:
I can't comment on that line itself, but while you're changing stuff, remove line 29. It's a comment for a function that no longer exists.